common/helm-repositories/rook-ceph.yaml

```diff
@@ -0,0 +1,10 @@
+---
+apiVersion: source.toolkit.fluxcd.io/v1beta1
+kind: HelmRepository
```

Comment:
I think we're missing adding this file into kustomization.yaml https://github.com/mesosphere/kommander-applications/blob/tga/add-ceph-operator/common/helm-repositories/kustomization.yaml
(I only noticed because I added a new app recently and forgot to add it too 😆)

Reply:
I was confused why it didnt got applied and i was punting debugging it while i was deploying it manually, thanks for catching it!